.dassie/config/initializers/hyrax.rb

```diff
@@ -5,6 +5,8 @@
   # Defaults to false.  See README for more info
   config.arkivo_api = true
 
+  config.analytics = ENV.fetch('HYRAX_ANALYTICS', 'false')
```

Comment:
i think this just removes the casting behavior from the default `Hyrax::Configuration` tooling.

```suggestion
```

Reply:
If this is removed then Hyrax.config.analytics just returns nil.

Reply:
if `Hyrax.config.analytics` returns `nil` then `Hyrax.config.analytics?` (which is the existing preferred usage) should return `false`.

the casting behavior should be something we do once, so we can avoid the string equality checks throughout.